Game.java

```diff
@@ -136,16 +255,40 @@ public String getInfo(Game newGame){
 	}*/
 }
 
-class bigPlayer{
+class Map{
+	String mapName = "";
```

Comment:
members should be private imo

Reply:
getters and setters are annoying for a POC, I would leave this as is because lombok can be used for production

Reply:
I like to hammer in conventions early to reinforce the habbit, but we can leave it as-is